Misc/NEWS.d/next/Core and Builtins/2022-09-21-14-38-31.gh-issue-96848.WuoLzU.rst

```diff
@@ -0,0 +1,3 @@
+Fix code parsing the :option:`-X int_max_str_digits <-X>` command line
```

Comment:
Rather than "Fix" perhaps describe what the undesirable behavior was that is now no longer happening?